tcms/management/models.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=import-outside-toplevel
```

Comment:
this doesn't look right. You should try moving the offending imports at toplevel and see if something fails or not.

Reply:
needs to be removed. this file needs more attention